errai-codegen/src/main/java/org/jboss/errai/codegen/meta/RuntimeEnum.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.errai.codegen.meta;
+
+/**
+ * @author Tiago Bento <tfernand@redhat.com>
+ */
+public class RuntimeEnum extends MetaEnum {
```

Comment:
Same as with Annotation, I think a more consistent name would be "JavaReflectionEnum".

Reply:
I'll rename this one too.